tests/cli/test_docs.py

```diff
@@ -296,3 +296,123 @@ def test_docs_list(
         my_caplog=caplog,
         click_result=result,
     )
+
+
+@pytest.fixture
+def context_with_site_built(titanic_data_context_stats_enabled_config_version_3):
+    context = titanic_data_context_stats_enabled_config_version_3
+    context.build_data_docs()
+    obs_urls = context.get_docs_sites_urls()
+    assert len(obs_urls) == 1
+    expected_index_path = os.path.join(
+        context.root_directory,
+        context.GE_UNCOMMITTED_DIR,
+        "data_docs",
+        "local_site",
+        "index.html",
+    )
+    assert os.path.isfile(expected_index_path)
+    return context
+
+
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+def test_docs_clean_no_site_specified_raises_helpful_error(
```

Comment:
Maybe one more test for `test_docs_clean_invalid_site_specified_raises_helpful_error`?

Reply:
This functionality doesn't exist in v2 at all. Let me make another task and PR to address this gap.